From 23e28dcedd781cd7d6b0e086ee2ef6095bec68aa Mon Sep 17 00:00:00 2001
From: Kazakov Stepan <kazakov12stepan2012@live.ru>
Date: Thu, 1 Nov 2018 21:54:45 +0300
Subject: [PATCH] build Lawnchair

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index eccf36f7..0164b21a 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -132,7 +132,7 @@ PRODUCT_PACKAGES += \
     messaging \
     OmniJaws \
     OmniSwitch \
-    PixelLauncher \
+    Lawnchair \
     SoundPickerPrebuilt \
     Stk \
     Terminal \
-- 
2.19.1

