From b6fd4b9314de466305f02dda93603c3c6de2dea0 Mon Sep 17 00:00:00 2001
From: Kazakov Stepan <ksrt12group@gmail.com>
Date: Sun, 11 Nov 2018 00:04:44 +0300
Subject: [PATCH] fix

Signed-off-by: Kazakov Stepan <ksrt12group@gmail.com>
---
 nci/jni/JcopManager.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/nci/jni/JcopManager.cpp b/nci/jni/JcopManager.cpp
index d4c320d2..53b114ef 100644
--- a/nci/jni/JcopManager.cpp
+++ b/nci/jni/JcopManager.cpp
@@ -75,10 +75,10 @@ void JcopManager::deleteInstance() {
 tNFA_STATUS JcopManager::JcopInitialize() {
   tNFA_STATUS wStatus = NFA_STATUS_OK;
   // Getting pointer to JCOP module
-  Pgpx_Jcop_handle = dlopen("system/lib64/libp61-jcop-kit.so", RTLD_NOW);
+  Pgpx_Jcop_handle = dlopen("system/lib64/libnqp61-jcop-kit.so", RTLD_NOW);
   if (Pgpx_Jcop_handle == NULL) {
     LOG(ERROR) << StringPrintf(
-        "%s: Error : opening (system/lib64/libp61-jcop-kit.so) !!", __func__);
+        "%s: Error : opening (system/lib64/libnqp61-jcop-kit.so) !!", __func__);
     return NFA_STATUS_FAILED;
   }
   // Getting pointer to ALA_Init function
@@ -183,7 +183,7 @@ tNFA_STATUS JcopManager::JcopDeInitialize() {
   if (Pgpx_Jcop_handle != NULL) {
     if ((dlclose(Pgpx_Jcop_handle)) != DL_STATUS_OK) {
       LOG(ERROR) << StringPrintf(
-          "%s: Error : closing (system/lib64/libp61-jcop-kit.so) !!", __func__);
+          "%s: Error : closing (system/lib64/libnqp61-jcop-kit.so) !!", __func__);
       return NFA_STATUS_FAILED;
     }
   } else {
-- 
2.19.1

